cmd/cmd.go

```diff
@@ -879,6 +879,24 @@ func NewCLI() *cobra.Command {
 		Args:    cobra.ExactArgs(0),
 		RunE:    RunServer,
 	}
+	serveCmd.SetUsageTemplate(serveCmd.UsageTemplate() + `
+Environment Variables:
+
+	OLLAMA_HOST
```

Comment:
This is also applicable to non-serve commands. It sets the remote those commands will communicate with

Reply:
@mxyng Correct. I was (and did not mention) I planned documenting OLLAMA_HOST for `run` that for another PR (this was for serve only). The semantics are different (e.g. client dial vs server binding).

Happy to roll it up into this PR if desired.

Reply:
It's fine to leave it for another change. I just wanted to make sure there's no confusion about the var